src/uct/api/uct.h

```diff
@@ -518,6 +522,7 @@ struct uct_md_attr {
     struct {
         size_t               max_alloc; /**< Maximal allocation size */
         size_t               max_reg;   /**< Maximal registration size */
+        size_t               max_private_data; /**< Max size of the md's private data */
```

Comment:
need to emphasize this is for sockaddr connection establishment
e.g max_conn_priv + update doc